module/workflow/src/Application/Form/DataTransformer/StatusIdDataTransformer.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Ergonode\Workflow\Application\Form\DataTransformer;
+
+use Ergonode\SharedKernel\Domain\Aggregate\StatusId;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+class StatusIdDataTransformer implements DataTransformerInterface
```

Comment:
IF i good remember we resignee from using DataTransfromers ... for simplest form models